dask/dataframe/core.py

```diff
@@ -6516,7 +6516,7 @@ def idxmaxmin_row(x, fn=None, skipna=True):
 
 
 def idxmaxmin_combine(x, fn=None, skipna=True):
-    if len(x) == 0:
+    if len(x) <= 1:
```

Comment:
Sorry for not getting to getting to this sooner. I think the `len(x) == 0` case was to handle empty DataFrames. Do you know if our test suite covers the `len(x) == 1` case? I'm somewhat surprised this works if we just pass a non-empty DataFrame through

Reply:
I think in the tests that were failing, the issue was that in the agg step, len(x) was occasionally == 1. This happened when split_every=2 and the total number of partitions was 3, so sometimes there was only one item in x.